src/codem/preprocessing/preprocess.py

```diff
@@ -547,39 +617,44 @@ def _calculate_resolution(self) -> None:
         """
         Calculates point cloud average point spacing.
         """
-        pdal_pipeline = [
-            self.file,
-            {"type": "filters.hexbin", "edge_size": 25, "threshold": 1},
-        ]
-        pipeline = pdal.Pipeline(json.dumps(pdal_pipeline))
+        tag = ["AOI", "Foundation"][int(self.fnd)]
+
+        pipeline = pdal.Reader(self.file)
+        pipeline |= pdal.Filter.hexbin(edge_size=25, threshold=1)
```

Comment:
Someday we should make that `edge_size` configurable. It's probably too coarse for drone collects and the like.

Reply:
Going to create a feature request issue from this comment...